package.json

```diff
@@ -31,21 +31,22 @@
     ]
   },
   "scripts": {
-    "prebuild": "rimraf build/ build-bin/ build-zip/ && patch-package",
-    "build": "npm run build:src && npm run build:bin",
+    "postinstall": "patch-package",
+    "prebuild": "rimraf build/ build-bin/",
+    "build": "npm run build:src",
     "build:src": "npm run prettify && npm run lint && npm run build:fast && npm run build:doc",
     "build:fast": "gulp build && tsc",
     "build:doc": "mkdirp doc/ && ts-node --type-check -P automation/tsconfig.json automation/capitanodoc/index.ts > doc/cli.markdown",
-    "build:bin": "ts-node --type-check -P automation/tsconfig.json automation/build-bin.ts",
-    "release": "npm run build && ts-node --type-check -P automation/tsconfig.json automation/deploy-bin.ts",
+    "build:standalone": "ts-node --type-check -P automation/tsconfig.json automation/run.ts build:standalone",
+    "build:installer": "ts-node --type-check -P automation/tsconfig.json automation/run.ts build:installer",
```

Comment:
`build:installer` builds the native executable oclif installers. Note that the "standalone zip packages" are not technically installers, they're just zip files that get extracted by the user to some folder. We do occasionally refer to the "standalone zip packages" as "standalone installation", though.